packages/angular-cli/blueprints/ng2/files/__path__/tsconfig.json

```diff
@@ -12,11 +12,6 @@
     "target": "es5",
     "typeRoots": [
       "../node_modules/@types"
-    ],
```

Comment:
the baseUrl and paths setup was actually removed from the blueprint tsconfig.json, beside the erroneous commit message. :smile: